### PR TITLE
net-vpn/i2pd-2.28.0 version bump

### PR DESCRIPTION
**Pull request by khumarahn**
copy previous ebuild, no changes

### CODE REVIEW

**Comment by khumarahn** [comment]
two weeks

ping

File: net-vpn/i2pd/i2pd-2.28.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd cmake-utils toolchain-funcs
```
Comment (juippis) [inline net-vpn/i2pd/i2pd-2.28.0.ebuild]: We usually leave one empty line below EAPI to follow default skel type, remember next time.

File: net-vpn/i2pd/i2pd-2.28.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+DOCS=( README.md contrib/i2pd.conf contrib/tunnels.conf )
+
+PATCHES=( "${FILESDIR}/${PN}-2.14.0-fix_installed_components.patch"
+	"${FILESDIR}/i2pd-2.25.0-lib-path.patch" )
+
+pkg_pretend() {
+	if tc-is-gcc && ! ver_test "$(gcc-version)" -ge "4.7"; then
+		die "At least gcc 4.7 is required"
+	fi
```
Comment (juippis) [inline net-vpn/i2pd/i2pd-2.28.0.ebuild]: Is this _really_ necessary? The profile defaults are way newer than 4.7...

Comment (khumarahn) [reply net-vpn/i2pd/i2pd-2.28.0.ebuild]: I am not sure. gcc-3.4 is still available and stable in the tree.

File: net-vpn/i2pd/i2pd-2.28.0.ebuild (hunk 1 above)
Comment (khumarahn) [reply net-vpn/i2pd/i2pd-2.28.0.ebuild]: ok!

File: net-vpn/i2pd/i2pd-2.28.0.ebuild (hunk 2 above)
Comment (juippis) [reply net-vpn/i2pd/i2pd-2.28.0.ebuild]: Yes but all the profiles default to newer ones. I dont even know what one needs to do to be able to use 4.7 with current toolchain. 

Btw I merged tihs already so please dont commit anything on this PR anymore :P

Comment (khumarahn) [reply net-vpn/i2pd/i2pd-2.28.0.ebuild]: ok, thanks!! I may remove the compiler check on next bump. It is exotic indeed.

**Pull request closed without merging**